openid-4-verifiable-presentations-1_0.md

```diff
@@ -283,6 +286,18 @@ The following is a non-normative example of an Authorization Request:
     &nonce=n-0S6_WzA2Mj HTTP/1.1
 ```
 
+The following is a non-normative example of a request object with a `create_request_uri``: 
+
+```
+{
+  "iss": "https://client.example.org",
+  "aud": "https://server.example.com",
```

Comment:
```suggestion
```

Reply:
why do you want to remove the audience?

Reply:
ok, it's signed.

please change client and server with some eloquent names belonging to this specific context and roles using Wallets ...